Plugins/ActSVG/include/Acts/Plugins/ActSVG/IndexedSurfacesSvgConverter.hpp

```diff
@@ -216,14 +189,17 @@ void convert(const GeometryContext& gctx, const surface_container& surfaces,
 /// @brief Unrolling function for catching the right instance
 ///
 /// @note parameters are as of the `convertImpl` method
-template <typename surface_container, typename tuple_type, size_t... I>
+template <typename surface_container, typename Head, typename... Tail>
 void unrollConvert(const GeometryContext& gctx,
                    const surface_container& surfaces, const Options& cOptions,
                    ProtoIndexedSurfaceGrid& sgi,
                    const Experimental::SurfaceCandidatesUpdator& delegate,
-                   const tuple_type& axesTuple, std::index_sequence<I...>) {
-  (convert(gctx, surfaces, cOptions, sgi, delegate, std::get<I>(axesTuple)),
```

Comment:
Why does the call from the DetectorVolumeFinderJsonConverter
```
(convert(jIndexedVolumes, delegate, detray, Args{}), ...);
```
not work here?

Reply:
Indeed, I could - well caught!